### ISSUE TITLE
Error while running on Raspberry Pi

### ISSUE THREAD

**Issue by tejapan** [calebolson123/BabySleepCoach#31]
I am a newbie in SW and been having trouble getting this up. I am running "docker compose up" but end up with the following error message - 

_> [baby-sleep-coach 6/8] RUN pip3 install -r requirements.txt:
2.105 ERROR: Could not find a version that satisfies the requirement mediapipe==0.9.1.0 (from versions: 0.10.3, 0.10.5, 0.10.7, 0.10.8)
2.107 ERROR: No matching distribution found for mediapipe==0.9.1.0
failed to solve: process "bin/sh -c pip3 install -r requirements.txt" did not completed successfully: exit code: 1_

I am running RaspberryOS 64-bit on RPi4b. Can anyone please help me with the above error.

**Comment by calebolson123** [comment]
Hmm does raspbian == RaspberryOS? I've got raspbian on my pi4b

**Comment by tejapan** [comment]
> Hmm does raspbian == RaspberryOS? I've got raspbian on my pi4b

Yeah, I think Raspbian is the old name. It is now known as RaspberryPiOS. I tried with the latest RaspberryPiOS(64-bit) released this Oct23rd and hit the same error. I then tried with legacy RaspberryPiOS(64-bit) release from May3rd and again hit the same error.
Can you tell me the version of the OS that you are running?

Looking up online I came across certain posts mentioning about Python3 version dependencies for mediapipe. Although Google's mediapipe documentation page shows that it is compatible from Python3.8-3.11.
Can you also share the Python3 version that you are using?

Thanks!

**Comment by calebolson123** [comment]
ive got a rpi 4b. I recall what I did now, and it's somewhat painful. I followed [this guide/installation](https://github.com/superuser789/MediaPipe-on-RaspberryPi) in order to get MediaPipe running on my pi. I might try to get a dockerized encapsulation of that installation in this repo for rpi users.. it's a pretty painful installation
